tests/processes/section_executor_test_files/ProcessingGlobalTestRawFileBear.py

```diff
@@ -0,0 +1,13 @@
+from coalib.bears.GlobalBear import GlobalBear
+from coalib.results.Result import Result
+
+
+class ProcessingGlobalTestRawFileBear(GlobalBear):  # pragma: no cover
```

Comment:
why no cover?
Add a comment above explaining it.
I am trying very hard (https://github.com/coala/coala/pull/4339) to remove "pragma: no cover", so only very specific exceptions are allowed.